xml/article_nfs_storage.xml

```diff
@@ -595,7 +595,7 @@ include /etc/drbd.d;</screen>
       Create a primitive to manage the NFS server daemon:
      </para>
 <screen>&prompt.crm.conf;<command>primitive nfsserver nfsserver \
-  params nfs_server_scope=SUSE nfs_shared_infodir="/var/lib/nfs"</command></screen>
+  params nfs_server_scope=SUSE</command></screen>
```

Comment:
Does the scope need to be removed too? I seem to recall that it was added as part of the same workaround. 

Or does it still serve a function on its own?

Reply:
Thanks Roger!